scripts/tools

```diff
@@ -176,12 +176,12 @@ tools_mirror()
     if
      (( ${rvm_force_flag:-0} == 1 ))
     then
-      __rvm_sed_i "${file}" -e "s/^ruby_${n}_url=.*$/ruby_${n}_url=http:\/\/www.mirrorservice.org\/sites\/ftp.ruby-lang.org\/pub\/ruby\/${n}/"
+      __rvm_sed_i "${file}" -e "s/^ruby_${n}_url=.*$/ruby_${n}_url=https:\/\/www.mirrorservice.org\/sites\/ftp.ruby-lang.org\/pub\/ruby\/${n}/"
```

Comment:
Note that https://www.mirrorservice.org/sites/ftp.ruby-lang.org/pub/ruby doesn't exist anymore, so I don't know how useful this code is in the first place.